src/app/add-task/add-task.component.ts

```diff
@@ -0,0 +1,48 @@
+import { NgFor } from '@angular/common';
+import { Component, OnInit, TemplateRef, ViewChild } from '@angular/core';
+import { FormBuilder, FormGroup, FormsModule, ReactiveFormsModule } from '@angular/forms';
+import { MatDialog } from '@angular/material/dialog';
+import { TaskListComponent } from '../task-list/task-list.component';
+import { TaskService } from '../sevices/task.service';
+import { Task } from '../models/task';
+
+@Component({
+  selector: 'app-add-task',
+  standalone: true,
+  imports: [ReactiveFormsModule, FormsModule, NgFor, TaskListComponent],
+  templateUrl: './add-task.component.html',
+  styleUrl: './add-task.component.scss'
+})
+export class AddTaskComponent implements OnInit {
+  typeOptions: string[] = [
+    "Story",
+    "Task"
+  ]
+  @ViewChild('addformtemplate') popupContent?: TemplateRef<any>;
+  formGroup!: FormGroup;
```

Comment:
non null assertion (!) please don't use this, instead you can define the variable like this:
formGroup: FormGroup | null = null;